core/src/package.rs

```diff
@@ -45,6 +50,62 @@ pub enum PackageImplementation {
     Native,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub enum ArgType {
+    #[serde(alias = "string")]
+    String,
+    #[serde(alias = "int", alias = "integer", alias = "i64")]
```

Comment:
Hmm, I get what you are saying but I don't agree; the value that is actually parsed from the string is an `i64`, and all values that may be parsed as `i64`s is accepted, serialised to JSON and sent to the package. It is indeed true that it is up to the parser in the package to treat it with the correct precision - and we don't have control over that - but what we do have control over is what we parse and what we send through JSON and that is indeed an `i64`.

On the package side, it is also useful to know that the value is guaranteed to fit into an `i64` as well. If they use a library where they may choose to parse it as an `i32` or `i64`, and knowing that we guarantee that the value fits into an `i64` but that we don't guarantee that it fits into an `i32` may be good. Regardless of whatever parser the package developers choose to use, the precision is up to that package and if they use `JSON.parse()` from JavaScript, it is going to be parsed into a number which has a known precision anyways. So, for languages without an `i64` type, developers will never expect that they actually get an `i64`, since it doesn't exist in the language and can thus not be used.

That was my way of reasoning about this. I can of course change this if you still think that it would be better to just say `int`, but there is a possibility that a Rust developer will see "Ah, ModMark promises that it is an integer, so it must be okay to parse as an i32" when that is in fact not the case.